packages/dashboard/src/components/resourceExplorer/components/panel.tsx

```diff
@@ -58,17 +60,19 @@ export const IotResourceExplorerPanel: React.FC<IotResourceExplorerPanelProps> =
   };
 
   const PanelCell = ({ item }: { item: ExtendedPanelAssetSummary }) => {
+    const isAlarm = !!(item?.type === 'AWS/ALARM');
```

Comment:
should this be some function we have tests around? I'm sure the type is pretty stable, but in a case where we want to extend other string values to be an alarm we'd want to have a place to define it

Reply:
I think we are using it multiple times in the component also?

Reply:
I talked to Norbert about this and he said this is the best way to verify that it's an alarm, because all alarms have this. So my understanding is this is a reliable indicator.

If we want to allow external alarms, that will require a much larger refactor, so I didn't try to attack that problem here.

Reply:
You're right, I can abstract it.

Reply:
Added abstracted function.